dts/bindings/clock/nordic,nrf52-hfxo.yaml

```
@@ -0,0 +1,30 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic nRF high-frequency crystal oscillator (nRF52 series)
+
+compatible: "nordic,nrf52-hfxo"
+
+include: fixed-clock.yaml
+
+properties:
+  clock-frequency:
+    const: 64000000
+
+  startup-time-us:
+    type: int
+    description: |
+      Startup time in microseconds. It corresponds to the twice the time
```

The startup time does not correspond to twice the time between TASKS_HFCLKSTART and EVENTS_HFCLKSTARTED, but this is a way of obtaining a safe approximation of the value.

I would suggest phrasing it like:
```suggestion
      Startup time in microseconds
      The value can be obtained by first measuring the time between
      TASKS_HFCLKSTART and EVENTS_HFCLKSTARTED. Then multiply
      the value by 2 to account for temperature and supply variations.
      
```